ote/src/clj/ote/services/transport.clj

```diff
@@ -33,15 +35,27 @@
                 (specql/columns ::transport-operator/transport-operator)
                 where {::specql/limit 1})))
 
-(def transport-services-passenger-columns
-  #{::t-service/id ::t-service/type :ote.db.transport-service/passenger-transportation
-    ::t-service/published? ::t-service/name})
+(def transport-services-columns
+  #{::t-service/id ::t-service/type
+    :ote.db.transport-service/passenger-transportation
+    :ote.db.transport-service/terminal
+    :ote.db.transport-service/rental
+    :ote.db.transport-service/brokerage
+    :ote.db.transport-service/parking
+    :ote.db.transport-service/created
+    :ote.db.transport-service/modified
+    ::t-service/published?
+    ::t-service/name})
 
 (defn get-transport-services [db where]
   "Return Vector of transport-services"
   (fetch db ::t-service/transport-service
-                transport-services-passenger-columns
-                where))
+                transport-services-columns
+                where
+                {::specql/order-by ::t-service/type
+                 ::specql/order-direction :desc
```

Comment:
put the next two lines of closing parens on this line

Reply:
Fixed